pkg/providers/common/common.go

```diff
@@ -147,3 +149,24 @@ func GetCAPIBottlerocketSettingsConfig(config *v1alpha1.BottlerocketConfiguratio
 
 	return strings.Trim(string(marshaledConfig), "\n"), nil
 }
+
+func GetExternalEtcdReleaseUrl(clusterVersion string, versionBundle *cluster.VersionsBundle) (string, error) {
+	clusterVersionSemVer, err := semver.New(clusterVersion)
```

Comment:
i don't think we need this as it is validated here: https://github.com/aws/eks-anywhere/blob/9a356ed3aac6f284b5d81eaa4aaf5dbd59c9811d/pkg/api/v1alpha1/cluster.go#L884 (this function is called in both cli and controller i think)